acos_client/tests/t.py

```diff
@@ -489,6 +504,8 @@ def run_all(ax, partition, pmap):
             port='80')
     except acos_client.errors.Exists:
         print("got already exists error, good")
+    except acos_client.errors.ACOSException:
```

Comment:
Needs response code - I'm actually pausing this review to write the patch so I can stop referencing it.

Reply:
I cleaned this up and rebased with PR #228